package/.prototools

```diff
@@ -0,0 +1,11 @@
+node = "20.12.1"
+npm = "10.5.0"
+extism-js = "1.0.0-rc9"
+
+[plugins]
+extism-js = "source:./extism-js.toml"
+
+# TODO: add a binaryen plugin - requires a more fine grain version interpolation though
```

Comment:
Mentioned this earlier on Discord, but never got around to adding the required `{version.major}` interpolation needed.